docs/reference/content/reference/connecting/authenticating.md

```diff
@@ -144,7 +144,7 @@ var principal = "drivers@KERBEROS.EXAMPLE.COM";
 var urlEncodedPrincipal = encodeURIComponent(principal);
 
 // Let's write the actual connection code
-MongoClient.connect(format("mongodb://%s@%s/kerberos?authMechanism=GSSAPI&gssapiServiceName=mongodb", urlEncodedPrincipal, server), function(err, db) {
+MongoClient.connect(f("mongodb://%s@%s/kerberos?authMechanism=GSSAPI&gssapiServiceName=mongodb", urlEncodedPrincipal, server), function(err, db) {
```

Comment:
this is actually legacy, we should just use string interpolation now:
```
MongoClient.connect(`mongodb://${urlEncodedPrincipal}@${server}/kerberos?authMechanism=GSSAPI&gssapiServiceName=mongodb`), function(err, db) {
```

Would you mind making this change in both places?